src/layman/user/rest_user.py

```diff
@@ -52,7 +52,13 @@ def delete(username):
         x_forwarded_items=x_forwarded_items,
         actor_name=username,
     )
-    delete_user(username)
-    return jsonify({
-        'username': username
-    }), 200
+    skipped_publications = delete_user_public_publications(username)
+    if not skipped_publications:
```

Comment:
I prefere the happy day scenario be the longist and raise error in the middle of the method.